scripts/template.go

```diff
@@ -1,22 +1,55 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
 package main
 
 import (
+	"flag"
 	"fmt"
-	"os"
+	"log"
 	"path/filepath"
 
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/template"
 	"github.com/elastic/go-ucfg/yaml"
 )
 
+// Flags
+var (
+	schemaDir string
+	version   string
+)
+
+func init() {
+	flag.StringVar(&schemaDir, "schema", "schemas/", "Schema directory containing .yml files.")
+	flag.StringVar(&version, "version", "", "ECS Version (required)")
+}
+
 func main() {
+	log.SetFlags(0)
+	flag.Parse()
+
+	if version == "" {
```

Comment:
Perhaps we the change around the version for the template and code cleanup to a separate PR to get in quickly?

Reply:
I was thinking that when I made this change, but then I figured it wasn’t a very important change so it could wait. But if you’d like I can pull this one out into its own PR?

Reply:
Would be great to have it seperate. I think it's important as it changes the template a bit.

Reply:
Created #279.